app/assets/stylesheets/govuk-component/_magna-charta.scss

```diff
@@ -0,0 +1,287 @@
+/*
```

Comment:
I think this file should be called `govspeak-magna-charta.scss` in the same way that the video enhancement that is also part of govspeak is name-spaced to it in `govspeak-enhance-youtube-video-links.js`